rio/Cargo.toml

```diff
@@ -68,14 +68,14 @@ wayland = [
 ]
 
 [package.metadata.deb]
-name = "rio"
+name = "rio-term"
 maintainer = "Raphael Amorim <rapha850@gmail.com>"
 copyright = "2023, Raphael Amorim <rapha850@gmail.com>"
 depends = "$auto"
 section = "admin"
 priority = "optional"
 assets = [
-    ["../target/release/rio", "usr/local/bin/", "755"],
+    ["../target/release/rio-term", "usr/local/bin/", "755"],
```

Comment:
Why `usr/local/bin`?

Reply:
It's already the default path, I didn't change it.